webapp/reviews/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.db import models
+
+class Review(models.Model):
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='reviews')
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='reviews')
+    anonymous = models.BooleanField(
+        'Anonymous')
+    rating = models.IntegerField(
+        'Rating',
+        null=True,
+        blank=True,
+        choices=[
```

Comment:
This should perhaps be a sliding value with integer increments between 1-5 (or maybe even 0-5).
This would be a more accurate representation of the data and reduce the maintenance of individual values.